src/main/java/org/skriptlang/skript/bukkit/loottables/elements/conditions/CondHasLootTable.java

```diff
@@ -0,0 +1,55 @@
+package org.skriptlang.skript.bukkit.loottables.elements.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.event.Event;
+import org.bukkit.loot.Lootable;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.loottables.LootTableUtils;
+
+@Name("Has Loot Table")
+@Description("Checks whether an entity or block has a loot table. The loot tables of chests will be deleted when the chest is opened or broken.")
+@Examples("if block has a loot table:")
+@Since("INSERT VERSION")
+public class CondHasLootTable extends Condition {
+
+	static {
+		Skript.registerCondition(CondHasLootTable.class,
+			"%entities/blocks% (has|have) [a] loot[ ]table",
+			"%entities/blocks% does(n't| not) have [a] loot[ ]table",
+			"%entities/blocks% (has|have) no loot[ ]table"
```

Comment:
I'm not sure how I feel about adding an extra pattern for `no`, it appears to be the only thing preventing this from being a PropertyCondition